src/javascript/app/common/form_validation.js

```diff
@@ -346,22 +346,23 @@ const Validation = (() => {
             }
 
             if (!field.is_ok) {
-                message = options.message || ValidatorsMap.get(type).message;
+                let message_template = options.message || ValidatorsMap.get(type).message;
                 if (type === 'length') {
-                    message = template(message, [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
+                    message_template = template('', [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
                 } else if (type === 'min') {
-                    message = template(message, [options.min]);
+                    message_template = template('', [options.min]);
                 } else if (type === 'not_equal') {
-                    message = template(message, [options.name1, options.name2]);
+                    message_template = template('', [options.name1, options.name2]);
                 }
                 all_is_ok = false;
+                field.$error.text(message_template);
```

Comment:
I think the initial idea about passing message to `showError` function as a parameter was fine. In that case we could move this line inside the function and just execute it before line 384. Do you think this would work?

Reply:
I think that would work indeed, I moved the line now into there.